### PR TITLE
Turn off Cypress recording/parallel runs

### PR DESCRIPTION
**Pull request by paulrosenzweig**
This is just a stopgap so we can trust CI again. It looks like Cypress will take about 25 minutes to run after this.

Once this is merged, we should merge `release-x.38.x` to `master`.

Also:

- Fixes for test failures now that we have separate EE/OSS versions of Oracle and Vertica drivers. These were likely legitimate failures, but we didn't notice them because Cypress tests weren't running
- Bump backend test HTTP client timeout on CircleCI to fix random test failures

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1298,7 +1298,7 @@ workflows:
       - fe-tests-cypress:
           matrix:
             parameters:
-              node: ["1", "2", "3", "4"]
+              node: ["1"]
               edition: ["ee", "oss"]
           name: fe-tests-cypress-<< matrix.node >>-<< matrix.edition >>
           requires:
```
Comment (camsaul) [inline .circleci/config.yml:1304]: I would still include the change to change `cypress-group` to include `ee`/`oss` here e.g.

```yaml
cypress-group: "default-<< matrix.edition >>"
```

I don't know if Cypress.io was smart enough to know they were different groups but I'd rather be safe than sorry. This way we don't forget to do it whenever we reenable Cypress

Comment (paulrosenzweig) [reply .circleci/config.yml:1304]: Yeah good call. It certainly doesn't hurt to include it!

**Review by camsaul — APPROVED** [review]

**Comment by paulrosenzweig** [comment]
`fe-tests-cypress-1-oss ` ran for 35 minutes and then failed with `Too long with no output (exceeded 10m0s): context deadline exceeded`. I just reran it.

**Comment by camsaul** [comment]
Yeah CircleCI is being EXTRA slow right now, I think it's a lot slower if you're look at it while it runs

**Comment by nemanjaglumac** [comment]
All green makes me happy. Let's give this a spin :)

**Pull request merged**